src/ifc-printer/main.cxx

```diff
@@ -6,6 +6,7 @@
 #include <fstream>
 #include "ifc/reader.hxx"
 #include "ifc/dom/node.hxx"
+#include <limits>
```

Comment:
Where is this needed in this file?

Reply:
It _was_ needed before, but the code was removed at your request. I simply forgot to remove the header as well.